File: test-files/swift/good-example.swift
import Foundation

class Calculator {
    private var result: Double = 0.0
    
    func add(_ a: Double, _ b: Double) -> Double {
        result = a + b
        return result
    }
    
    
    func divide(_ a: Double, _ b: Double) throws -> Double {
        guard b != 0 else {
            throw NSError(domain: "Calculator", code: 1)
        }
        result = a / b
        return result
    }
}

let calculator = Calculator()
print("Result: \(calculator.add(5, 3))")

File: test-files/dart/good-example.dart
import 'package:flutter/foundation.dart';

class Calculator {
  double _result = 0.0;

  double add(double a, double b) {
    _result = a + b;
    return _result;
  }

  double divide(double a, double b) {
    if (b == 0.0) {
      throw ArgumentError('Division by zero');
    }
    
    _result = a / b;
    return _result;
  }
}

void main() {
  final calculator = Calculator();
  debugPrint('Result: ${calculator.add(5, 3)}');
}